### ISSUE TITLE
Wonky looking notifications with firefox 46

### ISSUE THREAD

**Issue by bwat47** [johngraciliano/simplewhite#23]
With the simple white x theme, web notifications don't look very good (there's an ugly dropdown menu in the middle, odd placement of the X button, corners don't look rounded properly and it shows a space for an image on the left even when the notification doesn't display an image).

I've attached a screenshot of a notification with the simple white x theme and a notification with the default theme for comparison.

Default:
![default-notification](https://cloud.githubusercontent.com/assets/1442520/13904368/964f5354-ee74-11e5-934d-34b8ef62ecc8.png)

Simple White X
![simple-white-notification](https://cloud.githubusercontent.com/assets/1442520/13904367/964d9b22-ee74-11e5-81ea-d90ac5059ba4.png)

This is with firefox 46 beta 2

**Comment by johngraciliano** [comment]
Thank you for your report.  Can you list the steps of how to reproduce the problem?

**Comment by bwat47** [comment]
I used this test page for the screenshots: https://jsfiddle.net/yoshi6jp/Umc9A/
1. Click on Request permission
2. When firefox prompts, allow firefox to show notifications
3. Click the Show button to trigger a web notification

**Comment by johngraciliano** [comment]
I am working on it.  Thank you again, now also for providing me with an easy tool for diagnosis.

**Comment by johngraciliano** [comment]
The alerts box (`chrome://global/content/alerts/alert.xul`) layout was changed in Firefox 44, so the solution is past due.  Another important change was introduced with the current Firefox 45:  Now the box has a shadow because the window where it is rendered can have a transparent background.\*  I verified this in Windows 8.  I am working under the assumption this is also true for other platforms.
I should have a new version fixing and a few minor issues in a couple of days.

---

*The box position is different for because it is in a padded window.
